src/client/components/Map.js

```diff
@@ -27,4 +28,18 @@ function MapComponent({
   );
 }
 
+MapComponent.propTypes = {
+  centerMapCoordinates: PropTypes.array,
+  pathColor: PropTypes.string,
+  displayMarkers: PropTypes.bool,
+  zoom: PropTypes.number,
+  waypoints: PropTypes.arrayOf(
+    PropTypes.shape({
+      lat: PropTypes.number,
+      lng: PropTypes.number,
+      message: PropTypes.string
```

Comment:
Hmm... this is Niels sample component and in storybook he is actually passing in something called `waypointMessage` but he is destructuring message here. In any case it is not working and we don't plan to have messages on the waypoints themselves (I deliberately try to keep the map as simple as possible), so could you please just remove the message propType, remove where it is being destructured off the waypoint element (:22) and remove the data from the story in storybook?